src/common/utils/functions.ts

```diff
@@ -12,3 +12,14 @@ export function getAppLocation(sdk: Extension): string {
   }
   return locationName;
 }
+
+export const merge = (target: any, source: any) => {
```

Comment:
It is a function that combines two objects, target and source, by recursively merging their properties.

Reply:
@Contentstack-AnkitaD use lodash `merge` instead of creating your own. All these low level functions are available in libraries. Avoid recreating them
https://www.geeksforgeeks.org/lodash-_-merge-method/